from Utils.Exception.BaseException import BaseException


class SingletonException(BaseException):
    pass
